book.js

```diff
@@ -1,47 +1,56 @@
 let books = JSON.parse(localStorage.getItem('books')) || [];
-const bookListDiv = document.getElementById("book-list");
-const addBookForm = document.getElementById("add-book-form");
 
-// Define the functions to add and remove books
 function addBook(title, author) {
-  books.push({ title: title, author: author });
+  books.push({ title, author });
   localStorage.setItem('books', JSON.stringify(books));
 }
 
 function removeBook(title) {
-  books = books.filter(book => book.title !== title);
+  books = books.filter((book) => book.title !== title);
   localStorage.setItem('books', JSON.stringify(books));
 }
 
-// Define function to display book list on the top of the page
-
 function displayBooks() {
-  
-  bookListDiv.innerHTML = "";
-  for (let i = 0; i < books.length; i++) {
-    const book = books[i];
-    const bookElement = document.createElement("p");
-    bookElement.innerText = book.title + " by " + book.author;
-    bookListDiv.appendChild(bookElement);
-  }
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+  books.forEach((book) => {
+    const bookItem = document.createElement('div');
+    bookItem.classList.add('book');
+    bookItem.innerHTML = `<span>${book.title} by ${book.author}</span>`;
+    bookList.appendChild(bookItem);
+  });
 }
 
 displayBooks();
 
-//Create event listener when user Add book
-
-addBookForm.addEventListener("submit", function(event) {
-  
+const addBookForm = document.getElementById('add-book-form');
+addBookForm.addEventListener('submit', (event) => {
   event.preventDefault();
-
-  const titleInput = document.getElementById("title");
-  const authorInput = document.getElementById("author");
-  const title = titleInput.value;
-  const author = authorInput.value;
-
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
   addBook(title, author);
-  const bookElement = document.createElement("p");
-  bookElement.innerText = title + " by " + author;
-  const bookListDiv = document.getElementById("book-list");
-  bookListDiv.appendChild(bookElement);
+  displayBooks();
+  addBookForm.reset();
+});
+
+const removeBookForm = document.getElementById('remove-book-form');
+const removeTitle = document.getElementById('remove-title');
+books.forEach((book) => {
+  const titleOption = document.createElement('option');
+  titleOption.value = book.title;
+  titleOption.innerText = book.title;
+  removeTitle.appendChild(titleOption);
+});
+removeBookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const selectedTitle = removeTitle.value;
+  removeBook(selectedTitle);
+  displayBooks();
+  removeTitle.innerHTML = '';
+  books.forEach((book) => {
+    const titleOption = document.createElement('option');
+    titleOption.value = book.title;
+    titleOption.innerText = book.title;
+    removeTitle.appendChild(titleOption);
```

Comment:
- [ ] Great job on implementing the remove functionality, However, As you may notice on the wireframe, Kindly make sure you stick to the original design. the remove button is supposed to be on each created book and not the way you have implemented it.

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/235532791-54ac6cbd-f472-4a6a-aa08-b3a94a626da5.png)

### Original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/235532877-c5f90ace-6ebf-47b6-a9d2-0224b56feac3.png)